.github/workflows/tests.yml

```diff
@@ -119,24 +131,91 @@ jobs:
 
       - name: Start Covalent dispatcher server
         run: covalent start
+               
+      - name: Run SDK tests and measure coverage
+        if: ${{ steps.working-directory.outputs.sdk}} == 'true'
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 15
+          max_attempts: 2
+          on_retry_command: rm -f ~/.config/covalent/covalent.conf
+          command: |
+            PYTHONPATH=$PWD/ pytest -vv tests/covalent_tests  --cov=covalent --cov-config=.coveragerc
+            
+      - name: Generate SDK coverage report
+        run: coverage xml -o sdk_coverage.xml
```

Comment:
The coverage and upload steps also need to be conditional based on the `working-directory.outputs` value

Reply:
I have updated the steps to check for the respective values in `working-directory.outputs`